packages/core/admin/server/services/user.js

```diff
@@ -314,6 +314,15 @@ const displayWarningIfUsersDontHaveRole = async () => {
   }
 };
 
+/** Returns an array of interface languages currently used by users
+ * @returns {Promise<Array<string>>}
+ */
+const getLanguagesInUse = async () => {
+  const users = await strapi.query('admin::user').findMany({ select: ['preferedLanguage'] });
+
+  return users.map(user => user.preferedLanguage || 'en');
```

Comment:
By doing this if two users have the same language, it'll appear twice. Is it something we want?
If not we could either wrap it with a unique or create an aggregate with the number of users using a particular language (`{ languagesInUse: { en: 4, fr: 1, it: 2 } }`). WDYT?

Reply:
This is something intended, as this data format is easier to use later in the data analytics tool

Reply:
Alright :+1: